.buildkite/pipeline.yml

```diff
@@ -0,0 +1,25 @@
+plugin-node-cache: &plugin-node-cache
+  cache_node: true
+
+plugins: &plugins
+  - ssh://git@github.com/goodeggs/goodeggs-core-buildkite-plugin#v0.0.18: *plugin-node-cache
+
+steps:
+  - label: 'Install'
+    command: NODE_ENV=development yarn install --frozen-lockfile
+    plugins: *plugins
+
+  - wait
+
+  - label: 'All Tests'
+    command: yarn test
+    plugins: *plugins
+
+  - label: ':loudspeaker: Publish'
+    if: build.tag != null
+    command:
+      - export PATH="/tmp/ci-tools:./node_modules/.bin:$PATH"
+      - yarn publish
```

Comment:
I'm almost certain you'll need an "automation token" to publish ... the default one available on buildkite is read-only.  I think that means you'll also need an `.npmrc` file that references `NPM_TOKEN`.  I'll give you the value over Slack.